### PR TITLE
Post model

### PR DESCRIPTION
**Pull request by guragt**
Added Post model
Added factory for Post
Added tests for Post

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_201_007_142_822) do
+ActiveRecord::Schema.define(version: 20_201_021_131_027) do
```
Comment (Snick555) [inline db/schema.rb]: what is the strange format here?

File: spec/factories/posts.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :post do
+    description { Faker::Lorem.sentences(number: 1) }
+    image do
+      Rack::Test::UploadedFile.new(File.join(Rails.root, 'spec', 'files', 'post_image.jpg'))
+    end
+    commentable { true }
```
Comment (Snick555) [inline spec/factories/posts.rb]: you already have a default value in db

File: spec/factories/users.rb [hunk 3]
```diff
@@ -4,5 +4,15 @@
     nickname { Faker::Internet.unique.username }
     email { Faker::Internet.unique.safe_email }
     password { Faker::Internet.password }
+
+    factory :user_with_posts do
```
Comment (Snick555) [inline spec/factories/users.rb]: make it as a `trait :with_posts`

File: spec/factories/posts.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :post do
+    description { Faker::Lorem.sentences(number: 1) }
+    image do
+      Rack::Test::UploadedFile.new(File.join(Rails.root, 'spec', 'files', 'post_image.jpg'))
```
Comment (Snick555) [inline spec/factories/posts.rb]: for tests make the file as small as possible, for example 1px x 1px

File: app/uploaders/image_uploader.rb [hunk 5]
```diff
@@ -0,0 +1,50 @@
+class ImageUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
```
Comment (Snick555) [inline app/uploaders/image_uploader.rb]: please remove all commented code here

File: db/schema.rb (hunk 1 above)
Comment (guragt) [reply db/schema.rb]: I don't understand what you mean

**Review by Snick555 — APPROVED** [review]

**Pull request merged**